src/components/promotion/PromotionDetails.tsx

```diff
@@ -24,15 +23,17 @@ export default function PromotionDetails({
   description,
   expirationDate,
   restaurantName = 'Restaurant',
+  boldName = undefined,
+  boldDescription = undefined,
```

Comment:
I think you can remove the `= undefined` part because it should be undefined by default if that property isn't set on the Promotion object